server/pulp/server/util.py

```diff
@@ -182,3 +185,66 @@ def __init__(self, obj, filter=()):
         for k,v in obj.items():
             if k in filter:
                 self[k] = v
+
+def copytree(src, dst, symlinks=False, ignore=None):
```

Comment:
`shutil.copytree` accepts the `ignore_dangling_symlinks` keyword argument. The docs say:

```
If the file pointed by the symlink doesn't
exist, an exception will be added in the list of errors raised in
an Error exception at the end of the copy process.

You can set the optional ignore_dangling_symlinks flag to true if you
want to silence this exception. Notice that this has no effect on
platforms that don't support os.symlink.
```
I suppose we can add it later if we need it, though.